lib/Client.js

```diff
@@ -3996,6 +3996,20 @@ class Client extends EventEmitter {
         return this.requestHandler.request("POST", Endpoints.CHANNEL_TYPING(channelID), true);
     }
 
+    /**
+     * Set the status of a voice channel
```

Comment:
Probably worth noting that this will not work in stage channels since stage channel docs will inherit voice channel docs